docs/general_config.md

```diff
@@ -73,7 +74,8 @@ The following settings allow the user to control various timings/durations aroun
 "trialFeedbackDuration": 1.0,               // Time for user feedback between trials
 "sessionFeedbackDuration": 5.0,             // Time for user feedback between sessions
 "sessionFeedbackRequireClick" : false,      // Don't require a click to move past the scoreboard
-"defaultTrialCount" : 5,
+"defaultTrialCount" : 5,                    
+"maxPretrialAimDisplacement" : 0,           // Disable max pretrial aim displacement by default
```

Comment:
It's weird that 0, a value you'd naturally want to be able to put in to restrict to 0 motion, does the exact opposite and allows infinite motion. Might be better to use `-1` or any less than 0 value for this purpose, though that is also not quite sensible. `INF` if it is writable might be better.

Reply:
We can use a negative value to indicate this is off (not sure on Any serialization of infinity or nan). But, on related note, I would **highly** suggest against using a 0 value here, as it would mean if the mouse is aimed _any_ direction except identically at the 0 ray the trial would be invalid. So if the reference target has a size (which it must), you could hit the reference, but still almost always have an invalid trial. Even in the 1 pixel reference target case this could include possible floating point errors w/ representing 0 exactly.

Makes me think that we might want to include more in the documentation on suggestions for this value...

Reply:
What is the 0 direction then? Is that not where the user happened to aim to destroy the target? I was assuming this limit was based on the aim direction when the target was destroyed, not some other 0 direction.

Reply:
The 0 direction is the direction through the _exact center_ of the reference target (the canonical az/el = 0 direction up to the scene `spawnHeading` specified in a config). Another way to think of it is the 0 `eccH`/`eccV` value used to spawn task targets. 

Using this "true" 0 as opposed to where the player clicked on the reference avoids players "cheating" towards and edge of a reference target, then cheating again with pretrial movement. It also helps make task ID more uniform across trials.

Reply:
I see how a 0 entry wouldn't be useful. I do think it's a weird value to mean infinity. On a related note, would 180 or 360, or maybe both give the same behavior?

Reply:
I hadn't thought about large values here, but I suppose allowing up to 180° offset would basically give the full 360° view rotation as valid. This could also be an option.

I'm open to suggestions, but as implemented this value must be >0° for the pretrial aim displacement test to be applied.

Reply:
I like 180 more, which makes sense in physical sense. I can also imagine naive experimenters associating 0 with 'no displacement is allowed'. The 180 option will prevent that confusion.

Reply:
I realize that the way this angle is computed the aim displacement can exceed 180°. The math to decide on aim displacement is:

```
float viewDisplacement = sqrt(pow(az_displacement, 2) + (el_displacement,2))
```

So, if you produce an azimuth displacement of 180° (largest possible) then _any_ elevation displacement the resulting angle will be greater than a 180° threshold. I suppose 360° would be safe here instead, though maybe we want to calculate this angle differently.

Reply:
I'd suggest we change the math to make it a circular conic solid angle of the provided max displacement rather than using the current square cone. In other words, the orange in this diagram rather than red. 
![image](https://user-images.githubusercontent.com/26633010/135937811-d1958e8a-06ff-41a7-8398-270ec0da73f1.png)

- Image from [here](https://www.solarxy.org/2020/11/17/solar-radiation-the-fundamentals/)

Reply:
We were trying to use a circle. but may have done so with incorrect assumptions. 

The idea from  the code above is that the view displacement is given by the length/magnitude of the angular vector (i.e. [azim, elev]) pointing from the "ideal 0" view direction to the actual view direction. By thresholding the magnitude of this difference vector we establish a circular region in angle space over which we will allow view displacement. Thinking about this more I think its an incorrect approach to measuring the angle between two vectors... I think the usual approach is taking the arccos of the normalized dot product to get the angle between two vectors. I will look into moving to this approach shortly.

I believe the square region comes from thresholding both azimuth and elevation independently (which we are not doing). Having said that I think we may